tests/settings.py

```diff
@@ -10,6 +10,8 @@
 
 SECRET_KEY = 'secret'
 
+RESOLWE_API_HOST = 'localhost'
```

Comment:
> This can be fixed by using the IP of the docker bridge interface in development settings instead of localhost.

I guess here. Is there a quick way to read the IP of the docker bridge interface?